### PR TITLE
[FIX] Fix usage in Readme.md

### PR DESCRIPTION
**Pull request by hurrinico**
I have fixed readme.md file modifying path of the wizard button with the right one

### CODE REVIEW

File: base_location_geonames_import/README.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ system parameter.
 Usage
 =====
 
-Go to *Sales > Configuration > Address book > Localization > Import from Geonames*,
+Go to *Configurations > Technical > Cities/Locations Management > Import from Geonames*,
```
Comment (pedrobaeza) [inline base_location_geonames_import/README.rst]: s/Configurations/Configuration

Comment (tarteo) [inline base_location_geonames_import/README.rst]: Settings > Technical > Cities/Locations Management > Import from Geonames
Settings*

Comment (tarteo) [reply base_location_geonames_import/README.rst]: Settings > Technical > Cities/Locations Management > Import from Geonames
Settings*

**Review by tarteo — CHANGES_REQUESTED** [review]

File: base_location_geonames_import/README.rst (hunk 1 above)
Comment (pedrobaeza) [reply base_location_geonames_import/README.rst]: You are totally right! I was only correcting the put name without s, but it's not the English name.

**Comment by hurrinico** [comment]
@pedrobaeza @tarteo Fixed

**Review by tafaRU — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**